packages/contracts-bedrock/src/dispute/AnchorStateRegistry.sol

```diff
@@ -264,9 +260,6 @@ contract AnchorStateRegistry is Initializable, ISemver {
         }
```

Comment:
Not super related but for the sake of my OCD can you change this to

```
        if (!isGameClaimValid(game)) {
            revert AnchorStateRegistry_InvalidAnchorGame();
        }
```

Reply:
Done in 8e330bd